concepts/05 UI Components/DateBox/15 Specify Value Range.md

```diff
@@ -128,12 +130,64 @@ If an entered value falls out of the range, the UI component displays an error m
         // ...
     })
 
+##### Vue
+
+    <!-- tab: App.vue -->
+    <template>
+        <DxDateBox
+            :value="2015/12/31"
+            type="date"
+            min="2015/1/1"
+            max="2015/12/31"
+            date-out-of-range-message="Date is out of range"
+        />
+    </template>
+
+    <script>
+    import 'devextreme/dist/css/dx.common.css';
+    import 'devextreme/dist/css/dx.light.css';
+
+    import DxDateBox from 'devextreme-vue/date-box';
+    // ...
+    export default {
+        components: {
+            // ...
+            DxDateBox
+        }
+    }
+    </script>
+
+##### React
+
+    <!-- tab: App.js -->
+    import React from 'react';
+    import 'devextreme/dist/css/dx.common.css';
+    import 'devextreme/dist/css/dx.light.css';
+
+    import DateBox from 'devextreme-react/date-box';
+    // ...
+    class App extends React.Component {
+        render() {
+            return (
+                <DateBox
+                    value="2015/12/31"
```

Comment:
The `value` property will be changed from inside the component when a user selects another value. In uncontrolled mode (when you don't bind the `value` to a state variable and don't handle the value change event), such properties should be declared with the word `default`. More on this in the [Uncontrolled Mode](https://js.devexpress.com/Documentation/Guide/React_Components/State_Management/#Uncontrolled_Mode) article.

```suggestion
                    defaultValue="2015/12/31"
```